### PR TITLE
Use _.clamp

### PR DESCRIPTION
**Pull request by delphiactual**
Use _.clamp() to make clamping easier to read

### CODE REVIEW

**Review by ryan-rushton — APPROVED** [review]

File: src/locale/dim.json [hunk 1]
```diff
@@ -20,13 +20,13 @@
     "Beta": "BETA",
     "ButtonOff": "Inventory Mode",
     "ButtonOn": "Active Mode",
+    "ChangeFilter": "Change Filter",
     "Farming": "Recent Items",
     "FarmingStart": "Start Farming",
     "Loadouts": "Loadouts",
     "NoQuests": "You're not tracking any quests",
-    "Pursuits": "Pursuits",
-    "ChangeFilter": "Change Filter",
```
Comment (48klocs) [inline src/locale/dim.json:28]: Odd that this one wasn't reordered previously.

**Review by 48klocs — APPROVED** [review]
This is a bunch more readable. Nice.

File: src/app/utils/util.ts [hunk 2]
```diff
@@ -111,3 +111,13 @@ export function download(data: string, filename: string, type: string) {
...
 }
+
+/**
+ * Clamp `value` between `min` and `max`
+ * @param value value to be clamped
+ * @param min minimum allowed value
+ * @param max maximum allowed value
+ */
+export function clamp(value: number, min: number, max: number) {
+  return Math.min(max, Math.max(min, value));
```
Comment (bhollis) [inline src/app/utils/util.ts:122]: https://lodash.com/docs/4.17.15#clamp ?

File: src/app/item-triage/ValueDial.tsx [hunk 3]
```diff
@@ -1,10 +1,11 @@
 import { AppIcon, banIcon, tagIcon } from 'app/shell/icons';
+import { clamp } from 'app/utils/util';
 import React from 'react';
 import styles from './ValueDial.m.scss';
 
 /** expects a value from 1-100 but caps both ends */
 export default function ValueDial({ value }: { value: number }) {
-  value = Math.min(100, Math.max(0, value));
+  value = clamp(value, 0, 100);
```
Comment (delphiactual) [inline src/app/item-triage/ValueDial.tsx:8]: @nev-r should this have been?
```suggestion
  value = clamp(value, 1, 100);
```
 based on the comment above

File: src/locale/dim.json (hunk 1 above)
Comment (delphiactual) [reply src/locale/dim.json:28]: yeah, this happened when I tried running `yarn lint` to fix my problems with prettier not working in code

**Review by bhollis — APPROVED** [review]

**Pull request merged**